plugin/tests/apply_text_edits/replace_line_which_move.vader

```diff
@@ -0,0 +1,49 @@
+Include: utils.vader
+
+Given php (Original source):
+  <?php
+
+  class Foo
+  {
+      /**
+       * @var string
+       */
+      private $foo:
+
+      private function foo()
+      {
+          return $this->foo();
+      }
+  }
+
+Execute (Goes to the method line and changes the visibility):
+  10
+  call Apply(
+    \ Delete(10),
+    \ Insert(10, '    /**'),
+    \ Insert(11, '     * @var int $i'),
+    \ Insert(12, '     */'),
+    \ Insert(13, '    private function foo(int $i)')
```

Comment:
might be a stupid question but why don't we just have the above `Given` and then do `10d` and envoke the change visiblity, like wioth the `insert use` test?

```
Given php (source with no namespace):
  <?php
  
  class Foobar
  {
      public function foobar(Phpactor $phactor)
      {
      }
  }
Do (put the cursor over an existing class name and add use):
  /Phpactor\<Enter>
  :call phpactor#UseAdd()\<Enter>

Expect php (the use statement to be inserted):
  <?php

  use Phpactor\Phpactor;

  
  class Foobar
  {
      public function foobar(Phpactor $phactor)
      {
      }
  }
```

Reply:
Ah, ok, this isn't testing change visibility :)